ndvi_forecast/2_train.ipynb

```diff
@@ -563,13 +563,21 @@
   ],
   "metadata": {
     "kernelspec": {
-      "display_name": "Python 3",
+      "display_name": "Python 3.6 - AzureML",
```

Comment:
Is it Python 3.6 or 3.6.9

Reply:
Current Azure ML is 3.6.9 though they mention 3.6